packages/botonic-plugin-dialogflow/src/index.js

```diff
@@ -16,7 +22,7 @@ export default class BotonicPluginDialogflow {
         query: data,
         lang: lang,
         sessionId: sessionId,
-        v: this.options.version
+        v: version
```

Comment:
Try to put related logic together, or as close as possible. In this case I have to look at 3 different places in order to know what the value of `version` will be: 1) constructor sets  `defaultVersion`. 2) query() signature uses defaultVersion. 3) there's an `if` that sets `version` before axios call. 4) version is used in axios.

In this case, the module is very simple and it's not a problem. But when modules grow, this is one of the main reasons they become hard to read and to follow the logic.

Instead, we could just do:

`v: this.options.version || this.defaultVersion`